### PR TITLE
Remove jquery-ui part-5

### LINKED ISSUE #3010: Remove jquery-ui from batch 5 of scripts
Remove jquery-ui from following scripts:
interface/usergroup/facility_user.php
interface/usergroup/facility_user_admin.php
interface/usergroup/usergroup_admin.php
interface/weno/ajax_sample_code.php
interface/weno/confirm.php
library/custom_template/custom_template.php
library/custom_template/personalize.php

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3010 

A short video
https://www.loom.com/share/ee5520b8614a4f039a653d841d85dc97

### CODE REVIEW

**Comment by im-Amitto** [comment]
@bradymiller With this, I will be done with all the issues assigned to me in this repo and will move to flutter repo completely :)

File: library/custom_template/custom_template.php [hunk 1]
```diff
@@ -51,6 +51,14 @@ function listitemCode($strDisp, $strInsert)
...
 <head>
+<style>
+    .draggable {
+        cursor: pointer !important
+    }
+    .is-dragging {
+        cursor: move !important
+    }
+</style>
 <?php Header::setupHeader(['common', 'opener', 'jquery-ui', 'select2', 'ckeditor']); ?>
```
Comment (stephenwaite) [inline library/custom_template/custom_template.php:62]: hi @im-Amitto , does the above `jquery-ui` have to go as well?

Comment (im-Amitto) [reply library/custom_template/custom_template.php:62]: Thanks for the review. Forgot to remove it after removing the sortable function which was using it.

**Comment by bradymiller** [comment]
hi @im-Amitto , Code looks great and bringing into codebase. Thanks for the improvements! Here we come flutter!!

2DV

**Pull request merged**